Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Use latest stable (LTS) alpine releases
+
+# Stage 1: Build container
+FROM node:lts-alpine as builder
+
+# Install dependencies
+# Git and python are currently necessary to install some packages
+RUN apk update
+RUN apk upgrade
+COPY package*.json ./
+RUN apk add --no-cache --virtual .gyp git python make g++ bash
+RUN npm install
```

Comment:
Note the critical error. Let's definitely add `RUN npm audit fix` here. 
![screenshot_20190107_164507](https://user-images.githubusercontent.com/106148/50795255-9aed8900-129b-11e9-9de6-892b2ce782cc.png)

Reply:
@evalsocket I approved this by accident so please fix this before we merge

Reply:
Actually I just realized that there is no package-lock.json in the repo. Forget about adding `npm audit fix,` just update package.json and commit package-lock.json.

Reply:
Can you review it again i just change pkg manager from npm to yarn 
Please check [logs](https://hackmd.io/s/rJf4cU-G4)

Reply:
What advantages does changing from `npm` to `yarn` give us? I think we just need an updated package.json that doesn't have vulnerabilities while running `docker build` and we'll be good to go. 

We can also do that in another PR if it suits you.

Reply:
npm 5 is also faster so there is no point of discussion i revert the changes and in package-lock.json is added in repo.
**Npm audit result** -  There is a critical vulnerabilities command line injection in grunt-open@0.2.3 and no one is maintaining it. Some forks replace grunt-open by open. I think we can go with it for docker and create a new issue for removing warnings and vulnerabilities from package.json